### DISCUSSION TITLE
Is it possible to copy a collection?

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2862] [Archive]
Hi.

I was wondering if is possible to copy a collection for cases when we want to just add some extra fields or even duplicate one with a different name?

**Comment by benhaynes** [comment]
Not as of now, and I don't know how reliably we'd be able to do this (re: feature request). The problem is that any relational fields within the table would be broken, as we can't duplicate junction tables or O2M references. Trying to make a wizard to let you fix these issues during the duplication process seems messy, so unless we aligned on a good solution, this isn't on our roadmap as of yet.

**Comment by botzill** [comment]
Thx @benhaynes.

What about simple tables which do not have any relations(or simple relations), is that doable? What is I duplicate on DB level myself?

The idea is that we have different clients which need the same input data, but we create different tables so that we can easily separate that. Any idea of how can I make this, better? Can we have a single table that may be and each user will see its own records? 

Thx.

**Reply by benhaynes** [reply]
We try to avoid features that only work _sometimes_, so I'm not sure. Also, this could be scripted by custom logic pretty easily... read the table schema, and write a new one. You can open a Feature Request on the `directus/next` repo and see if this gets traction!
